docker-compose.yml

```diff
@@ -62,7 +62,7 @@ services:
       - "3001:3000"
     volumes:
       - type: bind
-        source: ./clients/privacy-center/config
+        source: ./data/config/privacy_center_config
```

Comment:
Use the custom privacy center config when running the privacy center via docker. This forces us to ensure we don't break this volume mount pattern!